### ISSUE TITLE
508-defect-4 [SCREENREADER]: Purely presentational down arrow icons SHOULD be hidden to screen reader users 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18704]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I do not want to hear purely presentational content.

## Details

Currently, the `<i>` down arrows in the "On this page" navigation are not hidden from screen reader users. This causes JAWS to announce links twice among other bugs (see screenshots).

## Acceptance Criteria

- [ ] Presentational content is not announced to the screen reader user

## Environment

* Mac, Safari/VO
* Windows, Chrome/JAWs

## Steps to Recreate

1. Enter `https://staging.va.gov/find-forms/` in Chrome with JAWs turned on
2. Using the `up` and `down` keys, navigate to the first link in the On this page navigation 
3. Confirm JAWs announces "Link, ..." first, and then upon a second `down` key, it announces "Link, ..." again

## Proposed Solution (if known)

Simply add `aria-hidden="true"` to the `<i>` down arrow

## Screenshots or Trace Logs

<img width="879" alt="Screen Shot 2021-01-18 at 5 42 07 PM" src="https://user-images.githubusercontent.com/14154792/104968849-fe474100-59b4-11eb-9b16-c6d11e8f02e0.png">

**Comment by DanielleThierryUSDSVA** [comment]
@SKasimow @youngfreezyVA This is another sitewide Drupal component issue.

**Comment by SKasimow** [comment]
@youngfreezyVA will fix for this page. if successful, we can bring it to CMS/PW team to see if they can deploy site wide.

**Comment by youngfreezyVA** [comment]
@joshkimux this would just be an `aria-hidden=true` on the icons right? that is what we have on the lang toggle link.

**Comment by joshkimux** [comment]
@youngfreezyVA Yes sir! I'll usually include those recommendations in the ticket under "proposed solution" 😄
